File: app-kotlin/src/main/java/com/tritiumstudios/phasmophobiaevidencetool/firebase/firestore/transactions/store/microtransactions/FirestoreMicrotransactions.java
package com.tritiumstudios.phasmophobiaevidencetool.firebase.firestore.transactions.store.microtransactions;

import androidx.annotation.NonNull;

import com.tritiumstudios.phasmophobiaevidencetool.firebase.firestore.transactions.store.FirestoreMarketplace;
import com.google.firebase.firestore.DocumentReference;

public class FirestoreMicrotransactions {

    private static final String
            DOCUMENT_MICROTRANSACTIONS = "Microtransactions";

    @NonNull
    public static DocumentReference getMicrotransactionsDocument() throws Exception {
        return FirestoreMarketplace.getStoreCollection()
                .document(DOCUMENT_MICROTRANSACTIONS);
    }

}

File: app-view/src/main/java/com/tritiumgaming/phasmophobiaevidencepicker/firebase/firestore/transactions/store/microtransactions/FirestoreMicrotransactions.kt
package com.tritiumgaming.phasmophobiaevidencepicker.firebase.firestore.transactions.store.microtransactions

import com.tritiumgaming.phasmophobiaevidencepicker.firebase.firestore.transactions.store.FirestoreMarketplace.Companion.storeCollection
import com.google.firebase.firestore.DocumentReference

class FirestoreMicrotransactions {
    companion object {
        private const val DOCUMENT_MICRO_TRANSACTIONS = "Microtransactions"

        val microTransactionsDocument: DocumentReference
            get() = storeCollection
                .document(DOCUMENT_MICRO_TRANSACTIONS)
    }

}
